examples/smartcore-ml/methods/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "methods"
+version = "0.1.0"
+edition = "2021"
+
+[build-dependencies]
+risc0-build = { version = "0.18.0" }
```

Comment:
We should use a combination of version and path so that if someone is developing locally, it will use the latest version of risc0. This is also useful for CI, it will detect when breaking changes in risc0 will cause an example to fail.

Reply:
I updated it to path so that it's aligned with the other examples.

Reply:
Also added the copyright and license blurbs to the necessary files where the checks were failing.